common/tier4_autoware_utils/include/tier4_autoware_utils/ros/published_time_publisher.hpp

```diff
@@ -91,7 +91,7 @@ class PublishedTimePublisher
   {
     bool operator()(const rmw_gid_t & lhs, const rmw_gid_t & rhs) const
     {
-      return std::memcmp(lhs.data, rhs.data, RMW_GID_STORAGE_SIZE) < 0;
+      return memcmp(lhs.data, rhs.data, RMW_GID_STORAGE_SIZE) < 0;
```

Comment:
```suggestion
      return std::memcmp(lhs.data, rhs.data, RMW_GID_STORAGE_SIZE) < 0;
```

And could you add:
`#include <cstring>` to the headers list?

- https://en.cppreference.com/w/cpp/string/byte/memcmp

Reply:
fixed with: https://github.com/autowarefoundation/autoware.universe/pull/6641/commits/1ddfa7b955962ac42dccb9cd6ddbe6a263b59dae